### ISSUE TITLE
Move environments into dedicated GCP projects

### ISSUE THREAD

**Issue by gnarea** [relaycorp/cloud-gateway#88]
For example, the Frankfurt environment should live in its own dedicated GCP project.

This is needed because Datastore/Firestore ties the whole project to a region 🤦🏾‍♂️ : https://cloud.google.com/datastore/docs/locations#selecting_a_location

**Comment by gnarea** [comment]
This needs to be added to the environment's module:

```hcl
resource "google_app_engine_application" "main" {
  // We don't actually use App Engine but this is the only way to enable Firebase Datastore
  project       = var.gcp_project_id
  location_id   = "europe-west3" // TODO: This should be environment-specific
  database_type = "CLOUD_DATASTORE_COMPATIBILITY"

  depends_on = [google_project_service.appengine]
}
```

**Comment by gnarea** [comment]
I fixed this in #89
